src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2115,6 +2115,9 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode, callable $
                                             ->integerNode('amount')->info('Amount of tokens to add each interval')->defaultValue(1)->end()
                                         ->end()
                                     ->end()
+                                    ->booleanNode('use_clock')
```

Comment:
why make this configurable?

Reply:
Do you prefer to always pass a clock when the component is installed?

Reply:
yes I would! One less option FTW

Reply:
works for me